GrandCentralBoard/Widgets/GitHub/View/GitHubCell.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  GitHubCell.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 21/05/16.
+//
+
+import UIKit
+
+
+struct GitHubCellViewModel {
+    let countLabelText: String
+    let nameLabelText: String
+    let color: UIColor
+}
+
+extension GitHubCellViewModel {
+
+    init?(forRepository repo: Repository) {
+        guard let prCount = repo.pullRequestsCount else {
+            return nil
+        }
+
+        countLabelText = "\(prCount)"
+        nameLabelText = repo.name.uppercaseString
+        color = UIColor.colorForPRCount(prCount)
+    }
+}
+
+private extension UIColor {
+    class func colorForPRCount(count: Int) -> UIColor {
+        switch count {
+        case 0 ..< 4: return UIColor(red: 246/255, green: 157/255, blue: 67/255, alpha: 1)
```

Comment:
Yes, but the red is different than one in GCBCore. I am not sure if it should change, when `gcb_fadedOrangeColor` in core is changed. But I will change it for now.